pkg/loader/util.go

```diff
@@ -171,6 +179,12 @@ func getColsOfTbl(db *gosql.DB, schema, table string) ([]string, error) {
 	if err = rows.Err(); err != nil {
 		return nil, errors.Trace(err)
 	}
+
+	// if no any columns returns, means the table not exist.
```

Comment:
IMHO, directly testing whether there's no `rows` returned for the specified schema and table name would be clearer.

Reply:
I think we should use https://godoc.org/database/sql#DB.QueryRow

Reply:
what do you means no rows return?
when there empty rows result.
will also return rows, and err = nil.
but the Next() will return false at first time you call it `rows.Next()`

Reply:
Is there any other cases when the table exists but `cols` is empty? If no, LGTM.

Reply:
no other cases